DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/Index.cshtml

```diff
@@ -19,15 +19,42 @@
   <div class="nhsuk-grid-column-full">
     <h1 class="nhsuk-heading-xl">Centre course setup</h1>
 
+    <div class="nhsuk-grid-row">

```

Comment:
Is column already in the VC?

Reply:
Yep, both the search-box VC and Sort have the column div inside. The following two have neither, hence why the row and column is present in the Index page.

Reply:
This is one of the things I noted to fix when fixing all the styling issues.

Reply:
Happy as long as it's documented on the ticket 👌